useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -19,11 +19,13 @@
 WHERE ContentBlock.Type = 'DataBlock'
   AND Releases.Published IS NOT NULL
   AND Releases.SoftDeleted = 0
-  -- Restrict DataBlocks that aren't featured tables or linked to content sections
-  AND (ContentSectionId IS NOT NULL
+  AND (
+    -- Include DataBlocks that are linked to Content Sections
+    ContentSectionId IS NOT NULL
+    -- Include DataBlocks that are linked to Content Sections
```

Comment:
I think you mean to say data blocks that are featured tables here.

Reply:
Oh yeah!

Updated, ta.  Hope you didn't mind the negating of the comments here - was easier on my brain.